### ISSUE TITLE
Breaks with Pyramid >= 1.6.x

### ISSUE THREAD

**Issue by fschulze** [Pylons/pyramid_skins#7]
The problem is the ``__get__`` method. Pyramid tries to figure out how to call the view and since 1.6.x it checks whether the view is an unbound method (it's in the backwards incompatibility list). That check causes an error, because ``inspect.isroutine`` returns ``True`` for the view because of the ``__get__`` method which triggers a ``True`` in ``inspect.ismethoddescriptor`` which is called by ``isroutine``.

Long story short: Can I just remove that feature? It's a pretty obscure one IMO.

The only other solution I see is changing Pyramid. It may be possible, that the ``isroutine`` call which calls 4 other methods could be replaces by one which skips the ``ismethoddescriptor`` call. I haven't investigated this option further yet.

**Comment by fschulze** [comment]
I've added a workaround in #8

**Comment by fschulze** [comment]
I released 2.0 which fixes compatibility.
